android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -94,6 +95,17 @@ private Boolean isTablet() {
     return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
   }
 
+  private Boolean isDeveloperModeEnabled() {
+    int isDeveloperModeEnabled = 0;
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+      isDeveloperModeEnabled = Global.getInt(this.reactContext.getContentResolver(), Global.DEVELOPMENT_SETTINGS_ENABLED, 0);
```

Comment:
we might use android.provider.Settings.Global.getInt() here instead of the import